cmake/GenerateHelptags.cmake

```diff
@@ -1,28 +1,36 @@
 if(DEFINED ENV{DESTDIR})
   file(TO_CMAKE_PATH
-    "$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc"
+    $ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc
     HELPTAGS_WORKING_DIRECTORY)
 else()
   file(TO_CMAKE_PATH
-    "${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc"
+    ${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc
     HELPTAGS_WORKING_DIRECTORY)
 endif()
 
 message(STATUS "Generating helptags in ${HELPTAGS_WORKING_DIRECTORY}.")
-if(EXISTS "${HELPTAGS_WORKING_DIRECTORY}/")
+if(EXISTS ${HELPTAGS_WORKING_DIRECTORY}/)
   message(STATUS "${HELPTAGS_WORKING_DIRECTORY} already exists")
-  # if the doc directory already exists, helptags could fail due to duplicate
+  # If the doc directory already exists, helptags could fail due to duplicate
   # tags. Tell the user to remove the directory and try again.
-  set(TROUBLESHOOTING "\nRemove \"${HELPTAGS_WORKING_DIRECTORY}\" and try again")
+  set(TROUBLESHOOTING "\nRemove \"${HELPTAGS_WORKING_DIRECTORY}\" and try again.")
 endif()
 
+# FIXME (fwalch):
```

Comment:
I'm not sure I'd call this a fixme... it looks like a decent and portable solution to me. :-)